src/brevitas_examples/imagenet_classification/ptq/imagenet_val.py

```diff
@@ -0,0 +1,332 @@
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import argparse
+from itertools import product
+import os
+import random
+import warnings
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.backends.cudnn as cudnn
+import torch.nn.parallel
+import torch.optim
+import torch.utils.data
+import torch.utils.data.distributed
+
+from brevitas import torch_version
+from brevitas.graph.calibrate import bias_correction_mode
+from brevitas.graph.calibrate import calibration_mode
+from brevitas.graph.quantize import COMPUTE_LAYER_MAP
+from brevitas.graph.quantize import quantize
+from brevitas.graph.target.flexml import FLEXML_COMPUTE_LAYER_MAP
+from brevitas.graph.target.flexml import quantize_flexml
+from brevitas.nn.quant_layer import QuantWeightBiasInputOutputLayer as QuantWBIOL
+from brevitas.nn.quant_mha import QuantMultiheadAttention
+from brevitas_examples.imagenet_classification.ptq.models import MODEL_LIST
+from brevitas_examples.imagenet_classification.ptq.models import model_with_cfg
+from brevitas_examples.imagenet_classification.ptq.models import ptq_quant_model_with_cfg
+from brevitas_examples.imagenet_classification.ptq.models import QUANT_MODEL_LIST
+from brevitas_examples.imagenet_classification.utils import accuracy
+from brevitas_examples.imagenet_classification.utils import AverageMeter
+from brevitas_examples.imagenet_classification.utils import generate_dataset
+from brevitas_examples.imagenet_classification.utils import SEED
+from brevitas_examples.imagenet_classification.utils import string_to_md
+from brevitas_examples.imagenet_classification.utils import transform_input
+
+LAYER_MAP = {'FP': COMPUTE_LAYER_MAP, 'Po2': FLEXML_COMPUTE_LAYER_MAP}
```

Comment:
we can spare a few more characters to describe things, call the fp_scale and po2_scale.